src/xdndworkaround.cpp

```diff
@@ -43,15 +43,17 @@
 
 #endif // (QT_VERSION >= QT_VERSION_CHECK(5, 4, 0))
 
-XdndWorkaround::XdndWorkaround():
-  lastDrag_(nullptr) {
```

Comment:
Yes, because it isn't defined for Qt < 5.4 -> https://github.com/lxde/libfm-qt/blob/master/src/xdndworkaround.h#L74